Wrapping/Generators/Python/CMakeLists.txt

```diff
@@ -171,11 +184,17 @@ if(NOT EXTERNAL_WRAP_ITK_PROJECT)
 
   # Install the package python files.
   set(ITK_WRAP_PYTHON_FILES )
+  set(ITK_WRAP_PYTHON_STUB_FILES )
   foreach(_file ${ITK_PYTHON_SUPPORT_MODULES})
     set(install_tgt "${CMAKE_CURRENT_SOURCE_DIR}/itk/${_file}.py")
     list(APPEND ITK_WRAP_PYTHON_FILES ${ITK_WRAP_PYTHON_FILES} "${install_tgt}")
   endforeach()
+  foreach(_file ${ITK_PYTHON_STUB_MODULES})
+    set(install_tgt "${CMAKE_CURRENT_SOURCE_DIR}/${_file}.py")
+    list(APPEND ITK_WRAP_PYTHON_STUB_FILES ${ITK_WRAP_PYTHON_STUB_FILES} "${install_tgt}")
+  endforeach()
   WRAP_ITK_PYTHON_BINDINGS_INSTALL(/itk/support "ITKCommon" ${ITK_WRAP_PYTHON_FILES})
+  WRAP_ITK_PYTHON_BINDINGS_INSTALL(/itk-stubs "ITKCommon" ${ITK_WRAP_PYTHON_STUB_FILES})
```

Comment:
`/itk-stubs` -> `/itk` to install the files in the package at `itk/__init__.pyi` and `itk/BaseClasses.pyi` (assuming this is where we want these files).